.github/workflows/ci.yml

```diff
@@ -119,6 +119,21 @@ jobs:
           args: --package dotenv-linter
         if: steps.cache-target.outputs.cache-hit != 'true'
 
+  build:
+    name: Build docker image
```

Comment:
Please move these lines [here](https://github.com/dotenv-linter/dotenv-linter/pull/458/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR296) without login and git tag.

Reply:
This will be removed and it is only there for seeing the result of the build, so if it is working.

https://github.com/dotenv-linter/dotenv-linter/pull/458#discussion_r779504676

Reply:
I would keep this step to check that everything works.
What do you think about it?